### PR TITLE
docs(review): review of documentation pages

### PR DESCRIPTION
**Pull request by nerda-codes**
I have reviewed the documentation pages for:
 
- Block Storage
- Cockpit
- Domains & DNS
- Secret Manager

I explained the different parts in a little bit more detail, and checked the grammar, typos, etc.

I haven't changed anything related to technical information.

### CODE REVIEW

File: docs/data-sources/secret_version.md [hunk 1]
```diff
@@ -5,82 +5,94 @@ page_title: "Scaleway: scaleway_secret_version"
...
-In addition to all arguments above, the following attributes are exported:
+This section lists the attributes that are exported by the `scaleway_secret_version` data source. These attributes can be referenced in other parts of your Terraform configuration:
 
-- `description` - (Optional) Description of the secret version (e.g. `my-new-description`).
-- `data` - The data payload of the secret version. more on the [data section](#data)
-- `status` - The status of the Secret Version.
-- `created_at` - Date and time of secret version's creation (RFC 3339 format).
-- `updated_at` - Date and time of secret version's last update (RFC 3339 format).
+- `description` - (Optional) The description of the secret version (e.g. `my-new-description`).
+- `data` - The data payload of the secret version. This is a sensitive attribute containing the secret value. Learn more in the [data section](#data).
```
Comment (Laure-di) [inline docs/data-sources/secret_version.md:93]: `- `data` - The data payload of the secret version. This is a sensitive attribute containing the secret value. Learn more in the [data section](#data-information)`

Comment (Laure-di) [reply docs/data-sources/secret_version.md:93]: This should fix the pipeline :)

**Review by Laure-di — CHANGES_REQUESTED** [review]

File: docs/data-sources/secret_version.md (hunk 1 above)
Comment (nerda-codes) [inline docs/data-sources/secret_version.md:93]:
```suggestion
- `data` - The data payload of the secret version. This is a sensitive attribute containing the secret value. Learn more in the [data section](#data-information).
```

**Review by Laure-di — APPROVED** [review]

File: docs/resources/secret_version.md [hunk 2]
```diff
@@ -30,36 +36,35 @@ resource "scaleway_secret_version" "v1" {
 
 The following arguments are supported:
 
-- `secret_id` - (Required) The Secret ID associated wit the secret version.
-- `data` - (Required) The data payload of the secret version. Must be no larger than 64KiB. (e.g. `my-secret-version-payload`). more on the [data section](#data)
+- `secret_id` - (Required) The ID of the secret associated with the version.
+- `data` - (Required) The data payload of the secret version. Must not exceed 64KiB in size (e.g. `my-secret-version-payload`). Find out more on the [data section](#data).
```
Comment (nerda-codes) [inline docs/resources/secret_version.md:40]:
```suggestion
- `data` - (Required) The data payload of the secret version. Must not exceed 64KiB in size (e.g. `my-secret-version-payload`). Find out more on the [data section](#data-information).
```

**Review by remyleone — APPROVED** [review]

**Review by remyleone — APPROVED** [review]

**Pull request merged**